packages/discovery-react-components/package.json

```diff
@@ -50,6 +50,8 @@
     "lodash": "^4.17.15",
     "mustache": "^3.1.0",
     "pdfjs-dist": "^2.2.228",
+    "react-error-boundary": "^1.2.5",
+    "react-resize-observer": "1.1.1",
```

Comment:
Did you investigate other libraries/code to handle what we need?

I'm a bit concerned about this library:
- not widely used
- hasn't been updated since Nov 2018
- no automated tests (there's a PR from 2018, but it's just languishing)
- is this still actively supported? (i.e. see lone PR from 2018)